### DISCUSSION TITLE
Is there any English G2P pretrained models available on speechbrain?

### DISCUSSION THREAD

**Discussion by JohnHerry** [speechbrain/speechbrain#2210] [Q&A]
We need a pretrained English G2P model for speechbrain,  the transcription can be cmudict format or  IPA format. Is there any resource?

**Comment by BenoitWang** [comment]
Hi @JohnHerry yes there is, you can check this one https://huggingface.co/speechbrain/soundchoice-g2p

**Comment by JohnHerry** [comment]
Thank you very much for the help！ I will try
